Xamarin.Forms.Controls/GalleryPages/TitleView.xaml

```diff
@@ -0,0 +1,24 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             x:Class="Xamarin.Forms.Controls.GalleryPages.TitleView"
+			 NavigationPage.TitleView="{StaticResource MyTitleView}">
+	
+	<ContentPage.Resources>
+		<ResourceDictionary>
```

Comment:
packing views in RDs is not something we want to encourage. I fixed the sample in the PR description accordingly. no need to actually fix the sample